src/JuMP.jl

```diff
@@ -199,6 +199,8 @@ end
 @Base.deprecate getNumVars(m::Model) MathProgBase.numvar(m)
 @Base.deprecate getNumConstraints(m::Model) MathProgBase.numlinconstr(m)
 
+getCost(m::Model) = m.redCosts
```

Comment:
Fair enough. You decide of course. I just thought that it's a nice convenience function. But please consider the getObjective function, as it is being exported but not implemented so far.